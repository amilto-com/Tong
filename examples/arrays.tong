let numbers = [1, 2, 3, 4, 5]
print("Numbers:", numbers)
print("Sum:", sum(numbers))
print("Length:", len(numbers))