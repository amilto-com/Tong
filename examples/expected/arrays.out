Numbers: [1, 2, 3, 4, 5]
Sum: 15
Length: 5
